api/repository/follow.go

```diff
@@ -33,8 +33,6 @@ func (c FollowRepository) GetFollowerCount(ID string) (count int, err error) {
 }
 
 func (c FollowRepository) GetFollowingCount(ID string) (count int, err error) {
-	return count, c.db.DB.Model(&models.Follower{}).Select("COUNT(follow_user_id)").Where("user_id = ?", ID).Find(&count).Error
-}
```

Comment:
@Jbholae did you removed it knowingly?

Reply:
@sawal-timsina No, I am unaware about this.

Reply:
@Jbholae Let's rollback this line. We need this as well.

Reply:
@sawal-timsina  As I've checked, this line of code is still there

Reply:
@Jbholae its been removed from line 36 🙇 Please check on your branch 
<img width="1122" alt="Screen Shot 2022-09-12 at 3 30 35 PM" src="https://user-images.githubusercontent.com/52816353/189623602-d6715ce8-8d09-48d3-976f-8084050773f4.png">